### ISSUE TITLE
[Bug]: When using F/E sync, the second time running a program, it won't, saying that there's a wineserver without F/E sync

### ISSUE THREAD

**Issue by KierPrev** [bottlesdevs/Bottles#2107]
### Describe the bug

After launching a program with bottles, and closing it, I must manually kill the wineserver, in order to run another program inside bottles, if I don't, they won't run.

EDIT: Only happens with Fsync or Esync enabled, don't happen with System.

### To Reproduce

1. Run a program with bottles, and close it
2. Try to run a second program, or the same, a second time. The program won't start.
3. Kill all the bwrap processes
4. Now programs can run again

### Package

Flatpak from Flathub

### Distribution

Ubuntu 22.04

### Troubleshooting Logs

```shell
Version: 2022.8.28-brescia-2
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: false
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.19.9-xanmod1
Distro:
    Name: GNOME
    Version: '43'
Disk:
    Total: 6249926656
    Free: 6249787392
RAM:
    MemTotal: 11.6GiB
    MemAvailable: 7.9GiB
Bottles_envs: null
```

### Display Protocol

X11

### Display Backend used by Bottles

X11

**Comment by KierPrev** [comment]
Solved by switching from soda to GE wine in my bottle.

**Comment by mirkobrombin** [comment]
I'm not able to reproduce this. Can you share your bottle.yml?

**Comment by KierPrev** [comment]
Hi, thank you! Yes, here it is:
[backup.txt](https://github.com/bottlesdevs/Bottles/files/9647888/backup.txt)

**Comment by KierPrev** [comment]
I updated the bug, apparently, it only happens when sync is enabled, because it detects a wineserver instance running without sync, and the program doesn't start.

**Comment by KierPrev** [comment]
`err:fsync:fsync_init Failed to open fsync shared memory file; make sure no stale wineserver instances are running without WINEFSYNC.`

**Comment by MYDIH** [comment]
Hi ! Same problem here ... 

It seems it's linked to some process not properly cleaned up I guess. I realized that after restarting my computer, the problem disapears, so the next time I had the issue, I ran `flatpack ps` and I saw that 3 processes were listed for `com.usebottles.bottles`. Killing those fixed the issue. After that, waiting carefully instead of force closing windows seems to prevent the problem to arise.

Hope it helps !
